src/core/server/deprecations/README.mdx

```diff
@@ -0,0 +1,206 @@
+---
+id: kibCoreDeprecationsService
+slug: /kibana-dev-docs/services/deprecations-service
+title: Core Deprecations service
+summary: The Deprecations service helps surface deprecated configs and features for plugins to our users
+date: 2021-06-27
+tags: ['kibana','dev', 'contributor', 'api docs']
+---
+
+# Core Deprecations service
+This guide is written in the format of a FAQ to help you get started using the deprecations service.
+For more details on the service contract we've documented all the service apis and properties under the
+deprecation service docs generated by in core.
+
+## What is the deprecations service?
+The deprecations service provides a way for the Kibana platform to communicate deprecated features
+and configs with its users. These deprecations are only communicated if the deployment is using these features,
+allowing for a user tailored experience for upgrading the stack version. The UA is the main consumer of the
+deprecations to show them to the users and potentially signal to Cloud that the Stack is ready for a major upgrade.
```

Comment:
[nit] I did open an issue to create a new status endpoint for cloud (https://github.com/elastic/kibana/issues/101282), so we can probably remove the word "potentially", as cloud should always check this endpoint to see whether or not all Kibana and ES deprecations are resolved.